app/components/navigation_header/search.tsx

```diff
@@ -37,11 +41,11 @@ const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => ({
 }));
 
 const NavigationSearch = ({
+    defaultHeight,
     largeHeight,
     scrollValue,
     hideHeader: setHeaderVisibility,
```

Comment:
This was forgotten by me. Do you mind changing the name of the function and also the definition? (it is defined as `(bool) => void` instead of `() => void`).

Reply:
done